aldryn_newsblog/migrations/0025_auto_20210421_0138.py

```diff
@@ -0,0 +1,275 @@
+        ),
+        migrations.AlterField(
+            model_name='newsblogconfig',
+            name='type',
+            field=models.CharField(max_length=100, verbose_name='Typ'),
+        ),
+        migrations.AlterField(
+            model_name='newsblogconfigtranslation',
+            name='app_title',
+            field=models.CharField(max_length=234, verbose_name='Name'),
+        ),
+        migrations.AlterField(
+            model_name='newsblogconfigtranslation',
+            name='language_code',
+            field=models.CharField(db_index=True, max_length=15, verbose_name='Sprache'),
+        ),
+        migrations.AlterField(
+            model_name='newsblogfeaturedarticlesplugin',
+            name='app_config',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='aldryn_newsblog.NewsBlogConfig', verbose_name='Apphook Konfiguration'),
+        ),
+        migrations.AlterField(
+            model_name='newsblogfeaturedarticlesplugin',
+            name='article_count',
+            field=models.PositiveIntegerField(default=1, help_text='Maximale Anzahl der "featured" Artikel.', validators=[django.core.validators.MinValueValidator(1)]),
+        ),
+        migrations.AlterField(
+            model_name='newsbloglatestarticlesplugin',
+            name='app_config',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='aldryn_newsblog.NewsBlogConfig', verbose_name='Apphook Konfiguration'),
+        ),
+        migrations.AlterField(
+            model_name='newsbloglatestarticlesplugin',
+            name='cache_duration',
+            field=models.PositiveSmallIntegerField(default=0, help_text='Wie lange (in sekunden) soll der Inhalt dieses Plugins im cache sein.'),
+        ),
+        migrations.AlterField(
+            model_name='newsbloglatestarticlesplugin',
+            name='exclude_featured',
+            field=models.PositiveSmallIntegerField(blank=True, default=0, help_text='Maximale Anzahl von "featured" Artikel, die ausgeschlossen sind. z.B. für Benutzer in Kombination mit "featured" Artikel Plugins.'),
+        ),
+        migrations.AlterField(
+            model_name='newsbloglatestarticlesplugin',
+            name='latest_articles',
+            field=models.IntegerField(default=5, help_text='Maximale Anzal der neusten Artikel.'),
+        ),
+        migrations.AlterField(
+            model_name='newsblogrelatedplugin',
+            name='cache_duration',
+            field=models.PositiveSmallIntegerField(default=0, help_text='Wie lange (in sekunden) soll der Inhalt dieses Plugins im cache sein.'),
+        ),
+        migrations.AlterField(
+            model_name='newsblogtagsplugin',
+            name='app_config',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='aldryn_newsblog.NewsBlogConfig', verbose_name='Apphook Konfiguration'),
+        ),
```

Comment:
All of this changes are not needed. They only change the translated texts.  
If the migration was generated with english as the default language this would not happen.

Reply:
I see. I will re-create the migration with English as the default language then. Thanks.